python/packages/streamsx/rest_primitives.py

```diff
@@ -33,6 +33,18 @@ def __init__(self, json_rep, rest_client):
     def __str__(self):
         return pformat(self.__dict__)
 
+    # Generically get elements from an object
+    # url - url of children
+    # json id in the returned json
+    # eclass - element class to create instances of
+    #
+    # Returns a list of eclass instances
```

Comment:
Should we use this style of doc? I feel like we should be consistent in doing the 

```
def my_function():
    """This is the function documentation
    """
    pass
```

style of documentation.